doc/conf.py

```diff
@@ -54,11 +54,14 @@
 
 # intersphinx configuration
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3/", None),
+    "contextily": ("https://contextily.readthedocs.io/en/stable/", None),
     "geopandas": ("https://geopandas.org/en/stable/", None),
     "numpy": ("https://numpy.org/doc/stable/", None),
+    "python": ("https://docs.python.org/3/", None),
     "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
+    "rasterio": ("https://rasterio.readthedocs.io/en/stable/", None),
     "xarray": ("https://docs.xarray.dev/en/stable/", None),
+    "xyzservices": ("https://xyzservices.readthedocs.io/en/stable", None),
```

Comment:
I haven't worked  with `intersphinx_mapping` before, but why are `rasterio` and `xyzservices` being added on this in addition to `contextily`? I'm assuming they're required dependencies for `contextily`?

Reply:
These intersphinx links are added so that ``` :doc:`rasterio <rasterio:index>` ``` and  ``` :class:`xyzservices.TileProvider` ``` can be used in the docstring.